frontend/components/KolideAce/KolideAce.jsx

```diff
@@ -15,6 +15,7 @@ class KolideAce extends Component {
   static propTypes = {
     error: PropTypes.string,
     fontSize: PropTypes.number,
+    handleSubmit: PropTypes.func,
```

Comment:
The standard across other files is to include a defaultProp if it's not required.

in this case
```js
import { noop } from 'lodash';

static defaultProps = {
    ...,
    handleSubmit: noop,
};
```